### PR TITLE
 sprint 4 yaml files

### PR DESCRIPTION
**Pull request by narasing01**
Please review the sprint 4 yaml files for the given files bgp_addition.txt,bgp_pe.txt,bgp_rr.txt and extcommunity_set.txt

### CODE REVIEW

File: gen/definitions/ext_community_set.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Extcommunity Set
```
Comment (danischm) [inline gen/definitions/ext_community_set.yaml:2]: Due to the YANG models having different lists for different types of extcommunity sets (opaque, cost, etc.) we would need to rename the resource to `Extcommunity Opaque Set`. Make sure you delete all the files created from the "old" name and then run `go generate` again.

File: gen/definitions/ext_community_set.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+path: Cisco-IOS-XR-um-route-policy-cfg:/routing-policy/sets/extended-community-opaque-sets/extended-community-opaque-set[set-name=%s]
+doc_category: Route Policy
+attributes:
+      - yang_name: set-name
```
Comment (danischm) [inline gen/definitions/ext_community_set.yaml:6]: Let's make sure the indentation in YAML files is consistent (2 spaces):

```
attributes:
  - yang_name: set-name
```

File: gen/definitions/router_bgp_address_family.yaml [hunk 3]
```diff
@@ -113,8 +115,9 @@ attributes:
       - yang_name: metric
         example: 100
 
+
 test_prerequisites:
-  - path: Cisco-IOS-XR-um-route-policy-cfg:routing-policy/route-policies/route-policy[route-policy-name=%s]
```
Comment (danischm) [inline gen/definitions/router_bgp_address_family.yaml:117]: Not sure why the indentation is changed here.

**Review by danischm — CHANGES_REQUESTED** [review]

File: gen/definitions/ext_community_set.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+        example: BLUE
+      - yang_name: rpl-extended-community-opaque-set
```
Comment (danischm) [inline gen/definitions/ext_community_set.yaml:8]: We should add a `tf_name: rpl` here.

**Review by danischm — CHANGES_REQUESTED** [review]

File: gen/definitions/ext_community_set.yaml (hunk 1 above)
Comment (narasing01) [reply gen/definitions/ext_community_set.yaml]: added the new yaml file for extcommunity_opaque_set

File: gen/definitions/ext_community_set.yaml (hunk 2 above)
Comment (narasing01) [reply gen/definitions/ext_community_set.yaml]: removed the extra space

**Comment by narasing01** [comment]
review comments incorporated

**Review by danischm — APPROVED** [review]

**Pull request merged**